docs/tls/tls-spans.md

```diff
@@ -0,0 +1,80 @@
+# Semantic Conventions for TLS spans
+
+**Status**: [Experimental](../../document-status.md)
```

Comment:
This link needs to be fixed to point at the specification.

Reply:
not applicable anymore due to the changes applied as suggested by @AlexanderWert